tunnel/intercept/protocols/tcp/conn.go

```diff
@@ -59,7 +59,7 @@ type ClientConn struct {
 	rcvAckNum   uint32
 	sndUna      uint32 // oldest unacknowledged sequence number
 	sndAckNum   uint32 // bytes received. written in tcpRecv. read in tcpSend
-	seqCond     sync.Cond
+	seqCond     *sync.Cond
```

Comment:
`go vet` was complaining that mutex's were being copied by value and not by reference -.-